modules/document_repository/README.md

```diff
@@ -0,0 +1,35 @@
+# Document Repository
+
+## Purpose
+
+ The Document Repository is a tool that provides studies with a centralized location for non-candidate documents. Files may be uploaded and organized under any user-defined category.
+
+## Intended Users
+ 
+ All Loris user and data entry staff could view, update, edit and delete the upload files with right permissions.
+
+## Scope
+
+ It's for all non-candidate documents.
+
```

Comment:
I think when you associate a Site to a document, only users from that Site can see it.   (If so, please definitely note this functionality in the spec.)